### PR TITLE
Adds download packet to methodology page (Sprint 4 designs)

### PR DESCRIPTION
**Pull request by vim-usds**
Looking for QA approval on download packet on the methodology page:

- [x] layout on desktop and mobile
- [x] validation of breakpoint transition (currently tablet)
- [x] appropriate text
- [x] overall QA

Thanks you

### CODE REVIEW

File: client/cypress/e2e/downloadPacket.spec.js [hunk 1]
```diff
@@ -2,22 +2,21 @@
 
 describe('Census Block Group download', () => {
   it('validate file download', () => {
-    // const filename = `usa.zip`;
-    cy.visit('localhost:8000/en/cejst');
+    const filename = `Screening+Tool+Data.zip`;
```
Comment (TomNUSDS) [inline client/cypress/e2e/downloadPacket.spec.js:5]: `+` in some filesystems can cause issues. Maybe convert them to something else?
see: https://www.mtu.edu/umc/services/websites/writing/characters-avoid/

File: client/src/pages/methodology.tsx [hunk 2]
```diff
@@ -21,37 +48,22 @@ const IndexPage = ({location}: MethodPageProps) => {
       </J40MainGridContainer>
 
       <J40MainGridContainer className={'j40-main-content'}>
-        <Grid row>
-          <Grid col>
+        <h1>{intl.formatMessage(messages.methodologyPageHeader)}</h1>
+        <Grid row gap>
+          <Grid col={12} tablet={{col: 6}}>
```
Comment (TomNUSDS) [inline client/src/pages/methodology.tsx:53]: nice

**Review by TomNUSDS — APPROVED** [review]
Only minor comments. LGTM.

File: client/cypress/e2e/downloadPacket.spec.js (hunk 1 above)
Comment (vim-usds) [reply client/cypress/e2e/downloadPacket.spec.js:5]: Hi @esfoobar-usds ,

How hard would it be to change filename to remove whitespaces?

Comment (esfoobar-usds) [reply client/cypress/e2e/downloadPacket.spec.js:5]: Very easy

Comment (vim-usds) [reply client/cypress/e2e/downloadPacket.spec.js:5]: @esfoobar-usds Great!

Should we do that now on this ticket or create a separate ticket?

**Pull request merged**